src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.foto {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
+  font-size: 16px;
```

Comment:
```suggestion

```

Reply:
font-size is redundant here. remove it according mentor comment

Reply:
I fixed it.